### PR TITLE
Doc updated for Industrial Edge Getting Started

### PR DESCRIPTION
**Pull request by claudiol**
- Updated the Getting Started section for Industrial Edge

### CODE REVIEW

**Comment by claudiol** [comment]
@abhatt-rh I do not get the spell error when I run `make test` locally on my machine.  
```$ make test
Running spellchecking on the tree
podman run -it -v /home/claudiol/work/gitrepos/docs:/tmp:"rw,z" docker.io/jonasbn/github-action-spellcheck:latest

Using pyspelling on configuration outlined in >.spellcheck.yml<
Checking files matching specified outlined in >.spellcheck.yml<
----------------------------------------------------------------
Spelling check passed :)
Ran all tests```

**Comment by claudiol** [comment]
NM I did find the issue.  Fixing the spell error.

File: content/patterns/industrial-edge/getting-started.md [hunk 1]
```diff
@@ -101,31 +149,45 @@ Install the installation tooling dependencies. You will need:
...
+   seldon-operator                manuela-ml-workspace
+   openshift-pipelines-operator-  openshift-operators
+   opendatahub-operator           openshift-operators
+   patterns-operator              openshift-operators
+   ```
+
+1. Access the ArgoCD environment
 
-1. Obtain the ArgoCD URLs and passwords
+   As a convenience the ArgoCD application links can be found in the OpenShift Applications Menu
```
Comment (abhatt-rh) [inline content/patterns/industrial-edge/getting-started.md:186]:
```suggestion
   You can find the ArgoCD application links listed under the **Red Hat applications** in the OpenShift Container Platform web console.
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @claudiol , I added a couple of comments for your consideration. Rest LGTM!

File: content/patterns/industrial-edge/getting-started.md [hunk 2]
```diff
@@ -101,31 +149,45 @@ Install the installation tooling dependencies. You will need:
...
 
 # Validating the Environment
 
 1. Check the operators have been installed
 
    ```text
-   UI -> Installed Operators
+   Operators > Installed Operators
    ```
+   The operators that should be installed on the HUB cluster are the following:
```
[the remark this answers is not in the document]
Comment (claudiol) [reply content/patterns/industrial-edge/getting-started.md]: @abhatt-rh Thank you so much for the assistance! All your suggestions have been applied.

**Pull request merged**